installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json

```diff
@@ -75,7 +75,10 @@
             "API": "build-pack-api",
             "LAMBDA": "build-pack-lambda",
             "WEBSITE": "build-pack-website"
-        }
+        },
+        "LAMBDA_SERVICES": ["cloud-logs-streamer", "deployments-event-handler", "environment-event-handler"],
```

Comment:
I see that few functions are missing here (that needs access to these components that  are in VPC now) - 

- `jazz_es-kinesis-log-streamer`